src/convertToObject.js

```diff
@@ -6,7 +6,24 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const splittedByDotComa = sourceString
+    .split(';')
+    .map((item) => item.trim())
+    .filter((item) => item.length > 0);
+
+  const splittedByDoubleDot = splittedByDotComa
+    .map((item) => item.split(':'))
+    .map((SplittedCssProperty) =>
+      // eslint-disable-next-line prettier/prettier
+      SplittedCssProperty.map((propertyPart) => propertyPart.trim()));
+
+  const cssProperiesObject = splittedByDoubleDot.reduce((acc, [key, value]) => {
```

Comment:
The variable 'cssProperiesObject' contains a typo. It should be 'cssPropertiesObject' to correctly reflect its purpose of storing CSS properties.

Reply:
typo fixed